docs/installation/windows.mdx

```diff
@@ -12,119 +12,34 @@ import Admonition from '@theme/Admonition';
-- **NPM**: Needed to manage packages.
-- **Windows Subsystem Linux (Ubuntu)**: Required to install for WSL2 installation.
-
-:::info
-When you install Cortex, all necessary dependencies for running it on your hardware, whether in CPU or GPU mode, will be automatically installed.
-:::
-
-### Hardware
-<Tabs>
-  <TabItem value="OS" label="OS" default>
-    - Windows 10 or higher.
-  </TabItem>
-  <TabItem value="CPU" label="CPU">
-    <Tabs>
-      <TabItem value="Intel" label="Intel" default>
-        <Admonition type="info">
-          - Jan supports a processor that can handle AVX2. For the full list, please see [here](https://en.wikipedia.org/wiki/Advanced_Vector_Extensions#CPUs_with_AVX2).
-          - We support older processors with AVX and AVX-512, though this is not recommended.
-        </Admonition>
-        - Haswell processors (Q2 2013) and newer.
-        - Tiger Lake (Q3 2020) and newer for Celeron and Pentium processors.
-      </TabItem>
-      <TabItem value="AMD" label="AMD">
-        <Admonition type="info">
-          - Jan supports a processor that can handle AVX2. For the full list, please see [here](https://en.wikipedia.org/wiki/Advanced_Vector_Extensions#CPUs_with_AVX2).
-          - We support older processors with AVX and AVX-512, though this is not recommended.
-        </Admonition>
-        - Excavator processors (Q2 2015) and newer.
-      </TabItem>
-    </Tabs>
-  </TabItem>
-  <TabItem value="RAM" label="RAM">
-    - 8GB for running up to 3B models (int4).
-    - 16GB for running up to 7B models (int4).
-    - 32GB for running up to 13B models (int4).
-      <Admonition type="info">  
-        We support DDR2 RAM as the minimum requirement but recommend using newer generations of RAM for improved performance.
-      </Admonition>
-  </TabItem>
-  <TabItem value="GPU" label="GPU">
-    - 6GB can load the 3B model (int4) with `ngl` at 120 ~ full speed on CPU/ GPU.
-    - 8GB can load the 7B model (int4) with `ngl` at 120 ~ full speed on CPU/ GPU.
-    - 12GB can load the 13B model (int4) with `ngl` at 120 ~ full speed on CPU/ GPU.
-    <Admonition type="info">  
-    - [NVIDIA driver](https://www.nvidia.com/Download/index.aspx) version 470.63.01 or higher.
-    - [CUDA Toolkit](https://developer.nvidia.com/cuda-toolkit) version 12.3 or higher.
-    </Admonition>
-  </TabItem>
-  <TabItem value="Disk" label="Disk">
-    - At least 10GB for app storage and model download.
-  </TabItem>
-</Tabs>
-
-## Cortex Installation
-
-### Step 1: Install Cortex
+## Overview
+For Windows, Cortex can be installed either directly on **Windows** or on **Windows Subsystem for Linux (WSL)**.
 
 :::info
-You can also install Cortex using the Cortex Installer available on [GitHub Releases](https://github.com/janhq/cortex/releases).
+If you're unsure about what is **Windows Subsystem for Linux (WSL)**, please refer to this [article](https://learn.microsoft.com/en-us/windows/wsl/faq).
```

Comment:
Just link within the previous statement? To have one less one info box (less clutter)